test/KnoBlockIO.ts

```diff
@@ -39,7 +39,7 @@ describe('KnoBlockIO contract', function () {
       it('Should add a new KnoBlock to the KnoBlock Mapping', async function () {
```

Comment:
```suggestion
      it('adds new KnoBlock to KnoBlock mapping', async function () {
```

Similarly in all other tests